omnigibson/envs/env_base.py

```diff
@@ -207,6 +206,10 @@ def _load_robots(self):
         """
         # Only actually load robots if no robot has been imported from the scene loading directly yet
         if len(self.scene.robots) == 0:
+            assert og.sim.is_stopped()
+            og.sim.play()
```

Comment:
Hmmm I would actually recommend importing the robots first before playing, since when the robot is loaded in it could be overlapping at (0, 0, 0)

Reply:
sorry what do you mean? in both cases, the robot could be overlapping at (0, 0, 0) right? Let's discuss in person.

Reply:
Try og.sim.pause()

Reply:
og.sim.pause does not work.